src/groups.js

```diff
@@ -81,6 +91,58 @@ class Group {
       ["owners", `${this.name}/owners`],
     ]);
   }
+
+
+  /**
+   * Policy for this Group itself.  Separate from the policy for the group's
+   * Source, which is the container for the group's datasets and narratives.
+   */
+  get authzPolicy() {
+    const {Read, Write} = authz.actions;
+    const {Type} = authz.tags;
+
+    const viewers = this.membershipRoles.get("viewers");
+    const editors = this.membershipRoles.get("editors");
+    const owners = this.membershipRoles.get("owners");
+
+    const basePolicy = [
+      /* All membership roles in a Nextstrain Group can see information about
+       * the group itself (e.g. members, role names, etc.), but only owners can
+       * update it.  Note that the datasets and narratives within the group are
+       * authz'd separately on the GroupSource.
+       */
+      {tag: Type.Group, role: viewers, allow: [Read]},
+      {tag: Type.Group, role: editors, allow: [Read]},
+      {tag: Type.Group, role: owners, allow: [Read, Write]},
+    ];
+
+    const publicPolicy = this.authzPublicPolicy();
+
+    return basePolicy.concat(publicPolicy);
+  }
+
+  /**
+   * A public policy for the group if appropriate.
+   * To be used in authzPolicy and the group's Source.
+   */
+  authzPublicPolicy() {
```

Comment:
I added this in 371efb9c77e4fc37f4644ae879a1a63a3a877434 but might be appropriate to add alongside d36f8441e0fd7d9d288cd92aa898caeb356b8a9d in #581

Reply:
Hmm. My first reaction is that we don't want to mark the Group with the Public authz tag because we want to condition members-only stuff like members and member roles on Read, even for public groups.

Also, it's likely ok for the management endpoints here for the overview/logo to be non-public even if the group's overview and logo are otherwise publicly viewable. Or we can change where authz is gated in the routing stack (e.g. like I noted in the comment I made a few minutes ago).

So I think there's a few ways to get from here to there, but all these intersecting bits may be easier to talk out in person/real time (though unfortunately it looks like I'll be out with a sick kiddo tomorrow).

Reply:
Ok, I just pushed 055efa733fdf52ea237b11f43198345afb242831 which removes authorization check on `/groups/:groupName`, and also dropped the test a1dec33b035e594e974a64a896fbe1534900eb9d since that will fail on authorization now (when I added it, it worked by using the public read access on `GroupSource` but we've moved to using `Group` as discussed in https://github.com/nextstrain/nextstrain.org/pull/570#discussion_r953081820).